### DISCUSSION TITLE
anti lc-hax?

### DISCUSSION THREAD

**Discussion by Amatheian** [winstxnhdw/lc-hax#440] [Q&A] [answered]
is there a way or codes when m hosting and no one can use hacks? someone joined my game and destroyed all the items i had, That was interesting move

**Comment by winstxnhdw** [comment]
Even if I do implement one, it wouldn’t be hard to work around it. There’s not much we can do here. Best thing you can do is have backup saves of your world always.

**Comment by winstxnhdw** [comment]
Maybe I can make a mod that helps you auto-backup your world.

**Reply by winstxnhdw** [reply, marked as answer]
Nvm, seems like it already exists: https://thunderstore.io/c/lethal-company/p/Larnetic/LethalCompanyBackupManager/
